tools/missing-test-detector/detector.go

```diff
@@ -68,7 +68,8 @@ func untestedFields(fieldCoverage ResourceChanges, path []string) []string {
 	for fieldName, coverage := range fieldCoverage {
 		if field, ok := coverage.(*Field); ok {
 			if !field.Tested {
-				fields = append(fields, strings.Join(append(path, fieldName), "."))
+				backtickedField := fmt.Sprintf("`%s`", strings.Join(append(path, fieldName), "."))
```

Comment:
This function should only return the field names to ensure separation of concerns: output formatting should all happen in one place.